opentreemap/treemap/util.py

```diff
@@ -225,3 +226,53 @@ def get_name_from_canonical_name(canonical_name):
 
 def make_udf_name_from_key(key):
     return 'udf:{}'.format(key)
+
+
+# Utilities for external links
```

Comment:
Consider moving all this code to a new `treemap/lib/external_link.py` file and keeping `util.py` for general-purpose utilities.